docs/openapi.yaml

```diff
@@ -279,6 +279,67 @@ paths:
         responses: { }
         httpMethod: POST
         type: AWS_PROXY
+
+  /{publicationIdentifier}/ticket/{ticketIdentifier}/update:
+    parameters:
+      - in: path
+        name: publicationIdentifier
+        schema:
+          type: string
+        required: true
+      - in: path
+        name: ticketIdentifier
+        schema:
+          type: string
+        required: true
+    put:
+      description: Update existing ticket
```

Comment:
I would add an elaborate description here explaining that only a few fields are exposed of the resource.